### DISCUSSION TITLE
Quarto - Customizing Table Output

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11423] [Blog]
# Quarto - Customizing Table Output

https://quarto.org/docs/blog/posts/2022-02-15-feature-tables/

**Comment by julianoxrj** [comment]
```{r}
#| tbl-cap: "Cars"
#| tbl-colwidths: [60,40]
#| tbl-cap-location: margin

library(knitr)
kable(head(cars))
```

**Comment by manuelreif** [comment]
The output of @tbl-i-created in html is ... 
"The output of table 1 in html is ..."
How can I change the default "table" to e.g. the german "Tabelle" or anything else?
Thanks!

**Reply by maikbode** [reply]
To change the default appearance of this *cross-reference*, add this to your projects (or documents) yaml-header:
```
crossref:
  fig-prefix: 'Abb.'
  tbl-prefix: 'Tab.'  # in your case change this to 'Tabelle', I used Tab. because I'm mostly citing inside brackets.
```

You can find more in [here on quarto](https://quarto.org/docs/authoring/cross-references.html#references).

**Comment by 0825kangkang** [comment]
is it possible to adjust the font size in the tables? thanks!
